Cargo.toml

```diff
@@ -5,6 +5,18 @@ version = "1.0.0"
 authors = ["Brett Cannon <brett@python.org>"]
 repository = "https://github.com/brettcannon/python-launcher"
 readme = "README.md"
+include = [
+    "Cargo.toml",
```

Comment:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields says `Cargo.toml` doesn't need to be listed.
```suggestion
```

Reply:
You're right, I'll remove that entry.